pkg/graph/schema.graphql

```diff
@@ -264,6 +264,159 @@ type CreateAccessibilityRequestDocumentPayload {
   userErrors: [UserError!]
 }
 
+"""
+The statuses for a system intake
+"""
+enum SystemIntakeStatusType {
+  """
+  Accepted
+  """
+  ACCEPTED
+  """
+  Business case needs changes
+  """
+  BIZ_CASE_CHANGES_NEEDED
+  """
+  Busness case draft
+  """
+  BIZ_CASE_DRAFT
+  """
+  Business case draft submitted
+  """
+  BIZ_CASE_DRAFT_SUBMITTED
+  """
+  Business case final needed
+  """
+  BIZ_CASE_FINAL_NEEDED
+  """
+  Business case final submitted
+  """
+  BIZ_CASE_FINAL_SUBMITTED
+  """
+  Intake is in draft
+  """
+  INTAKE_DRAFT
+  """
+  Intake is submitted
+  """
+  INTAKE_SUBMITTED
+  """
+  Lifecycle ID issued
+  """
+  LCID_ISSUED
+  """
+  Need business case
+  """
+  NEED_BIZ_CASE
+  """
+  Request not approved
+  """
+  NOT_APPROVED
+  """
+  Request is not an IT request
+  """
+  NOT_IT_REQUEST
+  """
+  Request requires no further governance
+  """
+  NO_GOVERNANCE
+  """
+  Request is ready for Governance Review Board meeting
+  """
+  READY_FOR_GRB
+  """
+  Request is ready for Governance Review Team meeting
+  """
+  READY_FOR_GRT
+  """
+  Request for shutdown of existing system is complete
+  """
+  SHUTDOWN_COMPLETE
+  """
+  Request for shutdown of existing system is in progress
+  """
+  SHUTDOWN_IN_PROGRESS
+  """
+  Request was withdrawn by business owner
+  """
+  WITHDRAWN
+}
+
+"""
+The request types for a system intake
+"""
+enum SystemIntakeRequestType {
+  """
+  Request is for major changes to an existing system
+  """
+  MAJOR_CHANGES
+  """
+  Request is for a new system
+  """
+  NEW
+  """
+  Request is for re-competing an existing system
+  """
+  RECOMPETE
+  """
+  Request is to shut down an existing system
+  """
+  SHUTDOWN
+}
+
+"""
+A SystemIntake instance
+"""
+type SystemIntake {
+  archivedAt: Time
+  businessNeed: String
+  businessOwner: String
+  businessOwnerComponent: String
+  component: String
+  contractEndMonth: String
+  contractEndYear: String
+  contractStartMonth: String
+  contractStartYear: String
+  contractVehicle: String
+  contractor: String
+  costIncrease: String
+  costIncreaseAmount: String
+  createdAt: Time!
+  decisionNextSteps: String
+  eaCollaborator: String
+  eaCollaboratorName: String
+  eaSupportRequest: Boolean
+  euaUserId: String!
+  existingContract: String
+  existingFunding: Boolean
+  fundingNumber: String
+  fundingSource: String
+  grbDate: Time
+  grtDate: Time
+  id: UUID!
+  isso: String
+  issoName: String
+  lcid: String
+  lcidExpiresAt: Time
+  lcidScope: String
+  oitSecurityCollaborator: String
+  oitSecurityCollaboratorName: String
+  processStatus: String
+  productManager: String
+  productManagerComponent: String
+  projectAcronym: String
+  projectName: String
+  rejectionReason: String
+  requestType: SystemIntakeRequestType!
+  requester: String
+  solution: String
+  status: SystemIntakeStatusType!
+  submittedAt: Time
+  trbCollaborator: String
+  trbCollaboratorName: String
+  updatedAt: Time!
+}
```

Comment:
I would break any set of fields with a shared prefix into a nested object. For example:

```gql
type SystemIntake {
  archivedAt: Time
  businessNeed: String
  businessOwner: SystemIntakeBusinessOwner
  ...
}

type SystemIntakeBusinessOwner {
  name: String
  component: String
}
```

I think we can do this for `businessOwner`, `contract`, `eaCollaborator`, `funding`,  `lcid`, `oitSecurityCollaborator`, `productManager`,  `project`, and `trbCollaborator`.

Reply:
We chatted as a group and will be working on a better structure once we have the front end switched over to use the new endpoint. Smaller changes, incrementally, rather than bigger changes all at once.